ambari-server/src/main/java/org/apache/ambari/server/orm/entities/UserAuthenticationEntity.java

```diff
@@ -74,9 +73,8 @@
   private UserAuthenticationType authenticationType = UserAuthenticationType.LOCAL;
 
   @Column(name = "authentication_key")
-  @Lob
   @Basic
-  private byte[] authenticationKey;
+  private String authenticationKey;
```

Comment:
Just assuming this isn't an existing column and one that was added for this feature (so that no database stuff has to happen in UpgradeCatalog)

Reply:
Correct. This is a new column for the feature. I was just adjusting the type since a CLOB or TEXT type was more relevant here.